backend/validators/loginValidator.js

```diff
@@ -0,0 +1,16 @@
+const { body } = require('express-validator');
+const params = {
+    minPasswordLength: 6,
+    maxPasswordLength: 32
+};
+
+module.exports = [
+    body('email').trim().not().isEmpty().withMessage('Email is required').isEmail().withMessage('Invalid email'),
```

Comment:
I think you can add also checking if email is already taken

Reply:
this comment should be for register validator

Reply:
done

Reply:
I like, when all validators are in separate lines

Reply:
:D I think it depend on line length and Prettier config